notes.go

```diff
@@ -0,0 +1,98 @@
+package chartmogul
+
+// Note is the customer note as represented in the API.
+type Note struct {
+	UUID string `json:"uuid,omitempty"`
+	// Basic info
+	CustomerUUID string `json:"customer_uuid,omitempty"`
+	Type         string `json:"type,omitempty"`
+	Text         string `json:"text,omitempty"`
+	Author       string `json:"author,omitempty"`
+	CallDuration uint32 `json:"call_duration,omitempty"`
+	CreatedAt    string `json:"created_at,omitempty"`
+	UpdatedAt    string `json:"updated_at,omitempty"`
```

Comment:
I don't think we need `omitempty` here as these values will always be there from the API (even if it's empty, it will be '' for `text` or `0` for `call_duration`). See [this article](https://www.sohamkamani.com/golang/omitempty/#when-to-use-omitempty)

But maybe my understanding is wrong